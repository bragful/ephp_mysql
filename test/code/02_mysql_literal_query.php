<?php

$db = mysqli_connect("127.0.0.1", "root", "root", "ephp_mysql");
$res = mysqli_query($db, "SELECT 100");
var_dump($res);
var_dump(mysqli_fetch_all($res, MYSQLI_NUM));
mysqli_data_seek($res, 0);
var_dump(mysqli_fetch_all($res, MYSQLI_ASSOC));
mysqli_data_seek($res, 0);
var_dump(mysqli_fetch_all($res, MYSQLI_BOTH));
mysqli_close($db);
