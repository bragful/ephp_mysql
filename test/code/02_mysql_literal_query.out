object(mysqli_result)#2 (5) {
  ["current_field"]=>
  int(0)
  ["field_count"]=>
  int(1)
  ["lengths"]=>
  array(1) {
    [0]=>
    int(3)
  }
  ["num_rows"]=>
  int(1)
  ["type"]=>
  int(0)
}
array(1) {
  [0]=>
  array(1) {
    [0]=>
    string(3) "100"
  }
}
array(1) {
  [0]=>
  array(1) {
    [100]=>
    string(3) "100"
  }
}
array(1) {
  [0]=>
  array(2) {
    [0]=>
    string(3) "100"
    [100]=>
    string(3) "100"
  }
}
